operator/pkg/tasks/init/wait.go

```diff
@@ -14,6 +14,8 @@ import (
 )
 
 var (
+	componentBeReadyTimeout = 120 * time.Second
```

Comment:
Could you explain why it's 120 seconds?

Reply:
By default, they are both set to 30 seconds. Many times, if an image needs to be pulled, it's not sufficient. In my environment, it probably needs to be set to 60 seconds. Of course, I cannot provide an exact timeout, but I believe 120 seconds should be sufficient for use in the vast majority of scenarios.

Reply:
Could you palease add some comments for the variable `componentBeReadyTimeout` ?

Reply:
ok